src/app/app.component.ts

```diff
@@ -7,4 +8,13 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'Susper';
+  constructor(private router: Router) { }
+  ngOnInit() {
+    this.router.events.subscribe((evt) => {
+      if (!(evt instanceof NavigationEnd)) {
```

Comment:
@nikhilrayaprolu I am not sure what you mean. This event is triggered, once a navigation action completes successfully. And will get triggered wherever I chose to change the route.
![image](https://cloud.githubusercontent.com/assets/20185076/25774034/dd7ab2ac-32a5-11e7-98a7-d122491d9c49.png)